honorspy.lua

```diff
@@ -133,6 +133,9 @@ local function StartInspecting(unitID)
         if (paused or (not C_PlayerInfo.UnitIsSameServer(PlayerLocation:CreateFromUnit(unitID)))) then
             return
         end
+        
+        -- Disable inspections while in a BG
+        if C_PvP.IsPVPMap() then return end
```

Comment:
This is a bit of an issue to me, being able to "register" players you meet in BG is kinda big on ERA as the population is quite low and having accurate data is not an easy thing